### PR TITLE
OMNI-2025-Fix-Pie-Chart-Stores

### PR DESCRIPTION
**Pull request by naresh130199**
updated and added pie chart and csv download feature

### CODE REVIEW

File: dashboard/src/App.css [hunk 1]
```diff
@@ -12,3 +12,44 @@
...
+  color: #fff;
+  text-align: center;
+  border-radius: 4px;
+  padding: 5px 0;
+  position: absolute;
+  z-index: 1;
+  bottom: 125%;
+  left: 50%;
+  margin-left: -60px;
+  opacity: 0;
```
Comment (omni-srejitk) [inline dashboard/src/App.css:35]: Refactor them and make it a component instead in the next PR

File: dashboard/src/routes/Analytics/Analytics.jsx [hunk 2]
```diff
@@ -113,22 +125,42 @@ export const Analytics = () => {
           storeData.set(saleData[1], saleData[2]);
         }
       });
-
-      let topStores = Array.from(storeData)
-        ?.sort((a, b) => +b[1] - +a[1])
-        ?.slice(0, 3);
-
+      let topStores = Array.from(storeData)?.sort((a, b) => +b[1] - +a[1]);
```
Comment (omni-srejitk) [inline dashboard/src/routes/Analytics/Analytics.jsx:128]: 1. Consider using more meaningful and intuitive names 
2. You don't need to create an array and filter it to check if it satisfices the condition
3. Remove the comments

**Review by omni-srejitk — APPROVED** [review]
Good work, Look into the comments suggested.

**Comment by omni-srejitk** [comment]
Feat : Fixed Top Stores Chart

File: dashboard/src/App.css (hunk 1 above)
Comment (naresh130199) [reply dashboard/src/App.css:35]: okay, will do that

**Pull request merged**